pkg/jx/cmd/install.go

```diff
@@ -1078,6 +1078,12 @@ func (options *InstallOptions) configureHelmValues(namespace string) error {
 		enableControllerBuild := true
 		helmConfig.ControllerBuild.Enabled = &enableControllerBuild
 	}
+
+	if options.Flags.Kaniko || options.Flags.Tekton {
+		// As we use Kaniko to build docker images, we don't need docker.sock bind mount in build containers
+		helmConfig.Jenkins.Agent.DockerHostPath = ""
```

Comment:
In case of `Tekton`, Jenkins installation should be disabled.

Reply:
Where are the tekton pod templates defined then ?

Reply:
There isn't any template. Typically, the volumes are mounted in the pipeline/tasks CRDs definitions. For Jenkins X, they are generated from jenkins-x.yaml file + buildpack file, which contains the high level pipeline definition.

Reply:
When I tried Tekton I got Task set with a bind mount for docker.sock. Not sure where this comes from.

Reply:
Yeah, that should be fixed now. I saw this as well, but in the latest version from yesterday, it doesn't seem to be the case.

Reply:
tested just now (1.3.1075) and still
```
Mounts:
      /builder/home from home (rw)
      /etc/podinfo from podinfo (ro)
      /home/jenkins from workspace-volume (rw)
      /home/jenkins/.docker from volume-0 (rw)
      /tools from tools (rw)
      /var/run/docker.sock from docker-daemon (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from tekton-bot-token-5thvv (ro)
      /workspace from workspace (rw)
```

Reply:
@warrenbailey @garethjevans Any idea why we are still mounting the docker socket? I saw in the prod cluster that it wasn't mounted.